server/models/instance.py

```diff
@@ -330,3 +328,5 @@ def finalizeInstance(event):
             msg += " for job(id={_id}, status={status})".format(**job)
             logger.debug(msg)
             Instance().updateInstance(instance)
+            notify_event([instance["creatorId"]], event,
```

Comment:
This doesn't fail cause `event` is already defined in this method (it's the main argument in the handler), but it isn't string.
My suggestion would be:
1) not override `event` but use something different like `event_name`
2) ensure it's always defined (currently bad things would happen for `status == JobStatus.QUEUED` etc.)